src/main/java/bowling_step3/domain/Frame/Frame.java

```diff
@@ -0,0 +1,25 @@
+package bowling_step3.domain.Frame;
+
+import bowling_step3.domain.Score;
+
+public abstract class Frame {
+
+    protected static final int DEFAULT = 0;
+    protected static final int MIN_PITCH_COUNT = 2;
+    protected static final int MAX_PITCH_COUNT = 3;
+
+    protected Score score;
+
+    public abstract void pitch(int countOfKnockDown);
+
+    public abstract boolean isFinish();
+
+    public abstract Frame next();
+
+    public abstract int getScore();
```

Comment:
```suggestion
    public int getScore() {
      return score.getScore();
    }
```
이렇게 추상클래스에서 메소드를 구현할 수 있습니다!
제가 제안드린 것은 이런식으로 공통 클래스는 추상 클래스에서 구현해도 된다는 이야기였습니다~

Reply:
아하... 제가 이해를 못했었네요 ŏ̥̥̥̥םŏ̥̥̥̥ 감사합니다 !!